### DISCUSSION TITLE
How to prevent HTML Body from from removing the script tag

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#11961] [Q&A] [answered]
Hi I want to use script tag and use javascript on the page, but whenever I save the page with script tag in HTML body the script tags are getting removed automatically while saving I want to prevent this from happening so that I can use javascript.

**Comment by Skrypt** [comment, marked as answer]
You can't add javascript to the Html Body because it is passed into an Html Sanitizer which removes these.
You need to add javascript by adding a LiquidPart to your Content-Type and then you will be able to add javascript.

Another option is to add an option to the HtmlSanitizer to allow for javascript but I'm really not recommending it.

**Reply by syedshadab19** [reply]
Hi @Skrypt thank for the response.  

can you also please let me know on how to add Another option to the HtmlSanitizer to allow for javascript which file to modify for it?because I am overwriting htmlbody with CK Editor so if I run into any problem I want to keep it as a backup.

**Reply by Skrypt** [reply]
See https://github.com/mganss/HtmlSanitizer

**Reply by syedshadab19** [reply]
@Skrypt Thanks You for the response.
